Week-1/Exercise_1.2/basefile.css

```diff
@@ -0,0 +1,171 @@
+body {
+    height: 100%;
+    border-right: 2px solid black;
+}
+
+
+body {
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
```

Comment:
.background_color__white {
background-color: #fff;
}

Then use it like this as class